src/server/game/Loot/LootMgr.cpp

```diff
@@ -144,7 +147,7 @@ uint32 LootStore::LoadLootTable()
 
         uint32 entry               = fields[0].GetUInt32();
         uint32 item                = fields[1].GetUInt32();
-        uint32 reference           = fields[2].GetUInt32();
+        int32  reference           = fields[2].GetInt32();
```

Comment:
@UltraNix have you checked that there are no entries that are too big for `int32` which previously fit in `uint32` ?

Reply:
But it's still 4 bytes. Only one entry now is signed.

Reply:
@UltraNix yes... which means `uint32` can hold bigger numbers than `int32` :D

Reply:
Nothing wrong happens here.